plugin/nvim-treesitter.vim

```diff
@@ -11,7 +11,7 @@ let g:loaded_nvim_treesitter = 1
 
 lua require'nvim-treesitter'.setup()
 
-highlight default TSNone term=NONE cterm=NONE gui=NONE
+highlight default TSNone term=NONE cterm=NONE gui=NONE guifg=foreground ctermfg=fg
```

Comment:
I'm getting an error with this change 

```
Error detected while processing .../nvim-treesitter/plugin/nvim-treesitter.vim:                                                                                                                                                 
line   14:                                                                                                                                                                                                                                      
E419: FG color unknown
```

Reply:
Hmm 🤔 so found an explanation for this in the vim docs
```
							*E419* *E420*
	When Vim knows the normal foreground and background colors, "fg" and
	"bg" can be used as color names.  This only works after setting the
	colors for the Normal group and for the Windows console.
```

I'm guessing from this that in your setup @stsewd you don't have the normal foreground set or maybe it's set dynamically or something. what does `hi Normal` output for you?

Reply:
I've tried adding a check to see if `Normal` foreground is set before using this now, can you see if that works for you?

Reply:
I have

```
:hi Normal                                                                                                              
Normal         xxx guifg=#D9D7CE guibg=#212733
```

The error is still present with your change